go.mod

```diff
@@ -1,6 +1,8 @@
 module github.com/rollkit/rollkit
 
-go 1.20
+go 1.21
```

Comment:
I go update necessary in this pr ?

Reply:
No, sorry, I'm just using 1.21 normally. I need to downgrade :/